app/views/root/_error_page.html.erb

```diff
@@ -0,0 +1,28 @@
+<%
+title = "Sorry, we are experiencing technical difficulties (#{status_code} error)" unless local_assigns.include?(:title)
+heading = "Sorry, we are experiencing technical difficulties" unless local_assigns.include?(:heading)
+intro = "Please try again in a few moments." unless local_assigns.include?(:intro)
```

Comment:
Is it worth moving these conditionals to the place they're rendered at, for clarity? Not a blocking change.